### DISCUSSION TITLE
Change member invitation email sending address

### DISCUSSION THREAD

**Discussion by scott-oneal** [community/community#110647] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello.  We have had our GitHub administrator resign, but when I send new member invites for our organization, it still sends the email from the previous admins email address.  How do we change the Sender address on invite emails?  Thanks.

**Comment by gustavostz** [comment, marked as answer]
You'll need to update the primary email address of the organization's current owner. Here's how you can do this:

1. Log in to GitHub as the current owner of the organization.
2. Go to Settings by clicking your profile picture in the top-right corner and selecting "Settings" from the dropdown menu.
3. Navigate to Emails in the left sidebar under "Account settings."
4. Add a new email address (if it's not already added) by entering it in the "Add email address" field and clicking "Add."
5. Verify the new email address by clicking the verification link sent to that email.
6. Set the new email address as primary by clicking the "Make primary" button next to it.

Once you've updated the primary email address, new member invites should be sent from this new address
